src/components/dialog/index.js

```diff
@@ -258,4 +251,25 @@ class Dialog extends BaseComponent {
   }
 }
 
+function createStyles(props) {
```

Comment:
you didn't remove the `createStyles` that inside the class
Also I think nothing is calling `setDialogSizeAndFlexType` anymore, so I wonder it's really needed?

Reply:
Not removed by mistake, removed.
Done.